databricks/koalas/internal.py

```diff
@@ -422,32 +422,82 @@ def attach_default_index(sdf):
         There are three types of default index that can be controlled by `DEFAULT_INDEX`
         environment variable.
 
-        - one-by-one: It implements an one-by-one sequence by Window function without
+        - sequence: It implements a sequence that increases one by one, by Window function without
             specifying partition. Therefore, it ends up with whole partition in single node.
             This index type should be avoided when the data is large. This is default.
+            See example below:
 
-        - distributed-one-by-one: It implements an one-by-one sequence by group-by and
-            group-map approach. It still generates a one-by-one sequential index globally.
-            If the default index must be an one-by-one sequence in a large dataset, this
+            >>> prev = os.environ.get("DEFAULT_INDEX", "sequence")
+            >>> os.environ["DEFAULT_INDEX"] = "sequence"
+
+            >>> ks.range(3).index
+            Int64Index([0, 1, 2], dtype='int64')
+
+            This is conceptually equivalent to the Spark example as below:
+
+            >>> from pyspark.sql import functions as F, Window
+            >>> spark_df = ks.range(3).to_spark()
+            >>> sequential_index = F.row_number().over(
+            ...    Window.orderBy(F.monotonically_increasing_id().asc())) - 1
+            >>> spark_df.select(sequential_index).rdd.map(lambda r: r[0]).collect()
+            [0, 1, 2]
+
+            >>> os.environ["DEFAULT_INDEX"] = prev
+
+        - distributed-sequence: It implements a sequence that increases one by one, by group-by and
+            group-map approach. It still generates the sequential index globally.
+            If the default index must be the sequence in a large dataset, this
             index has to be used.
             Note that if more data are added to the data source after creating this index,
             then it does not guarantee the sequential index.
+            See example below:
+
+            >>> prev = os.environ.get("DEFAULT_INDEX", "sequence")
+            >>> os.environ["DEFAULT_INDEX"] = "distributed-sequence"
+
+            >>> ks.range(3).index
+            Int64Index([0, 1, 2], dtype='int64')
+
+            This is conceptually equivalent to the Spark example as below:
+
+            >>> spark_df = ks.range(3).to_spark()
+            >>> spark_df.rdd.zipWithIndex().map(lambda p: p[1]).collect()
```

Comment:
Actually, RDD API is not used in actual index implementation but pandas group-map UDF is being used, and it mimics `zipWithIndex` in RDD API.